app/controllers/banneduser_experiment_controller.py

```diff
@@ -392,10 +392,15 @@ def _format_intervention_message(self, experiment_thing):
         condition = metadata_json["condition"]
         arm = metadata_json["arm"]
 
+        if condition not in self.experiment_settings["conditions"].keys():
```

Comment:
```suggestion
        if condition not in self.experiment_settings["conditions"]:
```

`key in dict()` and `key in dict().keys()` do the same thing here